GoogleUtilities/Environment/GULAppEnvironmentUtil.m

```diff
@@ -191,7 +191,7 @@ + (NSString *)deviceSimulatorModel {
 }
 
 + (NSString *)systemVersion {
-#if TARGET_OS_IOS
+#if TARGET_OS_IOS || TARGET_OS_VISION
   return [UIDevice currentDevice].systemVersion;
```

Comment:
Did you test this on VisionOS?

Reply:
What is the systemVersion?

Reply:
Yep, it returns `"1.0"` on the visionOS simulator at least. I'm not sure why we handle `TARGET_OS_TV` separately below since macOS and watchOS are the only platforms where [UIDevice](https://developer.apple.com/documentation/uikit/uidevice) isn't available -- need to download a tvOS simulator to check the behaviour there.

Reply:
We might have wanted to avoid the UIKit dependency but needed it for iOS because of backwards compatibility

Reply:
I updated it to use the `#elif` below since it works on visionOS as well and doesn't have the UIKit dependency.